packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json

```diff
@@ -816,6 +816,85 @@
                     }
                 }
             },
+            "DeployContractSolidityBytecodeJsonObjectV1Request": {
+                "type": "object",
+                "required": [
+                    "contractName",
+                    "contractAbi",
+                    "bytecode",
+                    "web3SigningCredential",
+                    "contractJSON",
+                    "constructorArgs"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "contractName": {
```

Comment:
@TonyRowntree Is `contractName` being used anywhere in the plugin's method that uses this request object? I couldn't find it.